ui/src/pages/Edit/UploadStory.tsx

```diff
@@ -0,0 +1,721 @@
+                name="summary"
+                label="Enter story summary"
+                defaultValue={formInput.summary}
+              />
+              <UploadLabelsText>Story</UploadLabelsText>
+              <StyledTextField
+                onChange={handleChange}
+                multiline
+                variant="outlined"
+                defaultValue={formInput.content}
+                placeholder="Lorem ipsum dolor sit amet, consectet ui i iadipiscing elit"
+                rows={8}
+                required
+                inputProps={{
+                  name: "content",
+                  id: "story-content",
+                }}
+                label="Enter story"
+              />
+            </StyledBackgroundColor>
+            <StyledBackgroundColor>
+              <UploadStoriesHeading>Author Information</UploadStoriesHeading>
+              <UploadLabelsText>First Name</UploadLabelsText>
+              <StyledTextField
+                onChange={handleChange}
+                variant="outlined"
+                required
+                id="author-first-name"
+                name="author_first_name"
+                label="Enter author's first name"
+                placeholder="Lorem ipsum"
+                defaultValue={formInput.author_first_name}
+              />
+              <UploadLabelsText>Last Name</UploadLabelsText>
+              <StyledTextField
+                onChange={handleChange}
+                variant="outlined"
+                required
+                id="author-last-name"
+                name="author_last_name"
+                label="Enter author's last name"
+                placeholder="Lorem ipsum"
+                defaultValue={formInput.author_last_name}
+              />
+              <UploadLabelsText>Biography</UploadLabelsText>
+              <StyledTextField
+                variant="outlined"
+                onChange={handleChange}
+                multiline
+                placeholder="Enter additional information here"
+                rows={4}
+                id="author-bio"
+                name="bio"
+                label="Enter author biography"
+                defaultValue={formInput.bio}
+              />
+            </StyledBackgroundColor>
+
+            <StyledBackgroundColor>
+              <UploadStoriesHeading>
+                Additional Information
+              </UploadStoriesHeading>
+              <FormControl>
+                <UploadLabelsText>Country of Origin</UploadLabelsText>
+                <StyledAutocomplete
+                  color="Primary"
+                  loading={!countries}
+                  options={countries || []}
+                  forcePopupIcon
+                  name="author_country"
+                  id="select-label-country"
+                  freeSolo
+                  // if user adds a new country display it as the default value, if not, look for prefilled value (edit stories)
+                  // prettier-ignore
```

Comment:
Why prettier-ignore?

Reply:
it looks ugly with prettier running on it lol, can delete the comment tho